src/internal/pfsdb/commits_test.go

```diff
@@ -707,3 +708,167 @@ func createBranch(ctx context.Context, t *testing.T, tx *pachsql.Tx, commit *pfs
 	_, err := pfsdb.UpsertBranch(ctx, tx, branchInfo)
 	require.NoError(t, err, "should be able to create branch")
 }
+
+func TestPickCommit(suite *testing.T) {
```

Comment:
I mentioned table tests below.  This is another place where they might be helpful.

You would create a type within this function body (which Go allows).
```go
type testCase struct {
  Name string
  Commits []*CommitInfo 
  Picker *CommitPicker
  Out *Commit
}

tcs := []testCase{

}
for _, tc := range tcs {
  t.Run(tc.Name, func(t *testing.T) {

 })
}
```

Reply:
I actually had considered doing a table test, but I think it would only be valuable for PickCommit(). However, for PickCommit, the picker cases require different set ups to best test them. Given that, I decided to just model each case as a function.